utils/users.js

```diff
@@ -140,6 +140,14 @@ function getUsernamesFromPRs(allPRs) {
   return usernames;
 }
 
+const checkRoleToUpdate = async (userData, newRoles) => {
+  const roles = { ...userData.roles };
+  const newRolesArray = Object.entries(newRoles);
+  if (roles[newRolesArray[0][0]] === newRolesArray[0][1]) return { updateRole: false };
```

Comment:
What is this doing? A comment here would be helpful.

Reply:
checking role already exist or not
role = {member:true}
ex: newRoleArray = [[member : true]]
if(roles[member]===true) return { updateRole: false };

Reply:
Isn't `{member:true}` an object?

```js
const role1 = {member: true};
const role2 = {member: true};

const boolResult = role1 === role2;
```

Is `boolResult` going to be true or false?

Reply:
return true if both objects are references to the exact same object. Otherwise, it will return false

Reply:
Done with changes.

Reply:
This will return false.
Ensure we are not following this way of testing the role